.github/milestone.yml

```diff
@@ -1,6 +1,6 @@
 base-branch:
-  - "luminous"
-  - "nautilus"
-  - "octopus"
-  - "pacific"
-  - "quincy"
+  - "(luminous)"
+  - "(nautilus)"
+  - "(octopus)"
+  - "(pacific)"
+  - "(quincy)"
```

Comment:
Not sure how this regex works, but could we make (if it makes sense) `master` match 'quincy' (whenever I see a `master` PR merged I can never directly guess which release it was eventually merged into).

Reply:
the action tries to match the head-branch with the regex, and if it matches, the matched group is used for the milestone. i cannot think of a regex which can make "master" match "quincy" at this moment.